core/src/main/java/org/geysermc/geyser/translator/text/MessageTranslator.java

```diff
@@ -422,6 +422,15 @@ public static String deserializeDescription(GeyserSession session, NbtMap tag) {
         return convertMessage(session, parsed);
     }
 
+    /**
+     * Deserialize an NbtMap with a description text component (usually provided from a registry) into a Bedrock-formatted string.
+     */
+    public static String deserializeDescriptionForTooltip(GeyserSession session, NbtMap tag) {
+        Object description = tag.get("description");
+        Component parsed = componentFromNbtTag(description);
+        return RESET + BASE + CharacterAndFormat.GRAY.character() + convertMessage(parsed, session.locale(), false);
```

Comment:
CharacterAndFormat can just use our ChatColor.GRAY.

Reply:
Addressed in latest commit